keadm/app/cmd/util/common.go

```diff
@@ -38,9 +38,10 @@ const (
 	DefaultDownloadURL = "https://download.docker.com"
 	DockerPreqReqList  = "apt-transport-https ca-certificates curl gnupg-agent software-properties-common"
 
-	KubernetesDownloadURL = "https://apt.kubernetes.io/"
-	KubernetesGPGURL      = "https://packages.cloud.google.com/apt/doc/apt-key.gpg"
-
+	KubernetesDownloadURL     = "https://apt.kubernetes.io/"
+	KubernetesGPGURL          = "https://packages.cloud.google.com/apt/doc/apt-key.gpg"
+	KubernetesBaseurl         = "https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64"
+	KubernetesGpgkey          = "https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg"
```

Comment:
Rename this as KubernetesGpgkey - >KubernetesCentOSGPGKey , Modify it where ever it is used.